lib/pure/os.nim

```diff
@@ -2009,28 +2009,32 @@ proc removeFile*(file: string) {.rtl, extern: "nos$1", tags: [WriteDirEffect], n
   if not tryRemoveFile(file):
     raiseOSError(osLastError(), file)
 
-proc tryMoveFSObject(source, dest: string): bool {.noWeirdTarget.} =
-  ## Moves a file or directory from `source` to `dest`.
+proc tryMoveFSObject(source, dest: string, isMoveDir: bool): bool {.noWeirdTarget.} =
```

Comment:
Sorry but what does `isMoveDir` mean?  ìsMovable`? `isDir`? `isMovableDir`(but then what are not movable directories)? Please use a clearer name.

Reply:
well it's a private proc, and `isMoveDir` is explained in the 1st line of comment; it means that it comes from `moveDir` as opposed to `moveFile`, a distinction which turned out to be needed for this fix.

how about `fromMoveDir` then so we can move forward with this PR?

Reply:
+1 for `isDir`, `tryMoveFSObject(source, dest, isDir = true)` looks semantically correct to me.

Reply:
ok isDir better, please go ahead with that

Reply:
agreed.